client/app/account/user/support-level/support-level.constants.js

```diff
@@ -0,0 +1,150 @@
+export const INACTIVE_SUPPORT_LEVEL = ['premium-accredited'];
+
+export const AVAILABLE_SUPPORT_LEVEL = ['standard', 'business', 'enterprise'];
+
+export const URLS = {
```

Comment:
maybe we could put and import those URLs from manager-core to avoid future code duplication

Reply:
This would imply exposing those constants as angular ones. I'm not sure it's what we want

Reply:
I'm on it @marie-j , I'm making a system that:
- has all constants
- you can request constants according to your language and subs, eta this week

Reply:
IMO, if theses URLS are only used here, they only should be here.